src/components/AttachmentPicker/index.native.tsx

```diff
@@ -109,37 +95,38 @@ const getDataForUpload = (fileData) => {
  * returns a "show attachment picker" method that takes
  * a callback. This is the ios/android implementation
  * opening a modal with attachment options
- * @param {propTypes} props
- * @returns {JSX.Element}
  */
-function AttachmentPicker({type, children, shouldHideCameraOption}) {
+function AttachmentPicker({type = CONST.ATTACHMENT_PICKER_TYPE.FILE, children, shouldHideCameraOption = false}: AttachmentPickerProps) {
     const styles = useThemeStyles();
     const [isVisible, setIsVisible] = useState(false);
 
-    const completeAttachmentSelection = useRef();
-    const onModalHide = useRef();
-    const onCanceled = useRef();
+    const completeAttachmentSelection = useRef<(data: FileObject) => void>(() => {});
+    const onModalHide = useRef<() => void>();
+    const onCanceled = useRef<() => void>(() => {});
+    const popoverRef = useRef(null);
```

Comment:
I think you should be able to remove this line

Reply:
I don't see `popoverRef` referenced anywhere. Can it be removed?